help/sources/tutorials/api/create/payments/stripe.md

```diff
@@ -0,0 +1,913 @@
+          "specName": "OAuth 2 Refresh Code",
+          "params": {
+            "accessToken": "{ACCESS_TOKEN}",
+          }
+      }
+  }'
+```
+
+| Property | Description |
+| --- | --- |
+| `name` | The name of your base connection. Ensure that the name of your base connection is descriptive as you can use this to look up information on your base connection. |
+| `description` | An optional value that you can include to provide more information on your base connection. |
+| `connectionSpec.id` | The connection spec ID of your source. This ID is fixed. The connection spec ID for [!DNL Stripe] is: `cc2c31d6-7b8c-4581-b49f-5c8698aa3ab3`. |
+| `auth.specName` | The authentication type that you are using to authenticate your source to Experience Platform. |
+| `auth.params.accessToken` | The access token of your [!DNL Stripe] account. Read the [[!DNL Stripe] authentication guide](../../../../connectors/payments/stripe.md#prerequisites) for steps on how to retrieve your access token. |
+
+**Response**
+
+A successful response returns the newly created base connection, including its unique connection identifier (`id`). This ID is required to explore your source's file structure and contents in the next step.
+
+```json
+{
+  "id": "a9950001-a386-4642-a0cd-5eaac6db5556",
+  "etag": "\"dc01244d-0000-0200-0000-65ea4e500000\""
+}
+```
+
+### Explore your source {#explore}
+
+Once you have your base connection ID, you can now explore the content and structure of your source data by performing a GET request to the `/connections` endpoint while providing your base connection ID as a query parameter.
+
+**API format**
+
+```http
+GET /connections/{BASE_CONNECTION_ID}/explore?objectType=rest&object={OBJECT}&fileType={FILE_TYPE}&preview={PREVIEW}&sourceParams={SOURCE_PARAMS}
+```
+
+**Request**
+
+When performing GET requests to explore your source's file structure and contents, you must include the query parameters that are listed in the table below:
+
+| Parameter | Description |
+| --------- | ----------- |
+| `{BASE_CONNECTION_ID}` | The base connection ID generated in the previous step. |
+| `objectType=rest` | The type of object that you wish to explore. Currently, this value is always set to `rest`. |
+| `{OBJECT}` | This parameter is required only when viewing a specific directory. Its value represents the path of the directory you wish to explore. For this source the value would be `json`. |
+| `fileType=json` | The file type of the file you want to bring to Platform. Currently, `json` is the only supported file type. |
+| `{PREVIEW}` | A boolean value that defines whether the contents of the connection supports preview. |
+| `{SOURCE_PARAMS}` | A [!DNL Base64-]encoded string that points to the resource path you want to explore. Your resource path needs to be encoded in [!DNL Base64] in order to obtain the approved format for `{SOURCE_PARAMS}`. For example, `{"resourcePath":"charges"}` is encoded as `eyJyZXNvdXJjZVBhdGgiOiJjaGFyZ2VzIn0%3D`. |
```

Comment:
Would it be useful to include all the `resourcePath`s available to use for stripe?